Tools/Scripts/build-and-analyze

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Apple Inc. All rights reserved.
```

Comment:
Nit:  Technically, this should be `2023-2024`.  Not sure if it matters in practice.